install_pim/manual/system_requirements/system_requirements.rst.inc

```diff
@@ -106,15 +116,9 @@ and the following PHP modules:
 **Search engine**
 
 +---------------+------------+
-| Elasticsearch | 6.5        |
+| Elasticsearch | 7.3        |
 +---------------+------------+
 
-and the following Java Runtime Environment version
```

Comment:
ES ships with its own JVM
so I guess it's totally useless to have that

Reply:
Are you sure about that ?

Reply:
Okay, they bundle a JDK in the standard download, but there's an option without JDK: https://www.elastic.co/downloads/elasticsearch-no-jdk

So maybe adding a remark about that: "Elasticsearch is bundled with a JDK, but if you want to use the no JDK version, please make sure to install the right JDK for Elasticsearch 7.x"